.github/workflows/at_libraries.yaml

```diff
@@ -53,3 +53,49 @@ jobs:
       - name: Analyze project source in at_server_status library
         working-directory: at_server_status
         run: dart analyze
+        
+      # Install dependencies in at_onboarding_cli
+      - name: Install dependencies in at_onboarding_cli
+        working-directory: at_onboarding_cli
+        run: dart pub get
+      
+      # Run dart analyzer in at_cli_onboarding
+      - name: Run dart analyzer in at_onboarding_cli
+        working-directory: at_onboarding_cli
+        run: dart analyze
+      
+      #functional tests for at_onbaording_cli
+      - name: add entry to hosts file
+        run: echo echo "127.0.0.1    vip.ve.atsign.zone" | sudo tee -a /etc/hosts
+        
+      - name: Get functional test dependencies
+        working-directory: at_onboarding_cli/functional_tests
+        run: dart pub get
+        
+      - name: Start docker instance
+        working-directory: at_onboarding_cli/functional_tests
+        run: sudo docker-compose up -d
+        
+#       - name: Check for docker container readiness
+#         working-directory: at_onboarding_cli/functional_tests
+#         run: dart run check_docker_readiness.dart
+
+      - name: ping
+        run: ping -c 10 vip.ve.atsign.zone -p 64
+        
+      - name: run pkamLoad on docker-image
+        run: sudo docker exec functional_tests_virtualenv_1 supervisorctl start pkamLoad
+      
+#       - name: Check test environment readiness
+#         working-directory: at_onboarding_cli/functional_tests
+#         run: dart run check_test_env.dart
```

Comment:
This does need the PKAM load to succeed (and won't work if the name resolution is messed up).